File: Programs/python/python_prac/const.py
class car():
    def __init__(self,name,age,gmail):
        self.name=name
        self.name=age
        self.gmail=gmail
        print("I am inside Car constructor")
        
        
class Innova(car):
    def __init__(self,name,age,gmail):
        super().__init__(name,age,gmail)
        print("I am inside Innova constructor")
        print(f'My name is {self.name} with age = {self.gmail}')
        
class Swift(car):
    def __init__(self,name,age):
        self.name=name
        self.age=age
        print(f'My name is {self.name} with age = {self.age}')
        print("I am inside Swift constructor")
        
obj = Swift("syed",29)
obj = Innova("Tousif",20,"syed@tousif")

File: Programs/python/python_prac/const.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{

class Car
{
    public:
    Car()
    {
        cout<<" I am in constructor\n";
    }
};

class Innova:public Car
{
    public:
    Innova()
    {
        cout<<" I am inside Innova class\n";
    }
};

class Swift:public Car
{
    public:
    Swift()
    {
        cout<<" I am inside Swift class\n";
    }
};

//Car object;
Innova innova_1;
Swift swift_1;
}